src/patternfly/sass-utilities/mixins.scss

```diff
@@ -152,3 +152,70 @@
   scrollbar-width: none; // hides scrollbars in Firefox 64 and up
   -ms-overflow-style: -ms-autohiding-scrollbar; // auto hides scrollbars in Edge
 }
+
+@mixin pf-hidden-visible($val: "block") {
+  &.pf-m-hidden {
```

Comment:
I feel like we should make a note that you dont want to use these modifiers on cells like inside of the expandable content. This is what happened when I added those modifiers and we run into the problem where hiding the cell doesnt hide those styles that come with the expansion.

<img width="687" alt="Screen Shot 2019-04-18 at 10 08 59 AM" src="https://user-images.githubusercontent.com/20118816/56366999-4c1c8d00-61c2-11e9-9c69-bfd978391714.png">

Reply:
nice, I'll look at that

Reply:
I tried this out and it seems to work OK for me. It may have been an issue with how it was written previously.